### ISSUE TITLE
Rice distribution is misdefined

### ISSUE THREAD

**Issue by nbud** [pymc-devs/pymc#3286]
## Description of your problem

The Rice distribution ``Rice(nu, sigma)`` is the distribution of ``sqrt(X^2+Y^2)`` where ``X~N(nu, sigma^2)``  and ``Y~N(nu, sigma^2)``, with X and Y independent. This parametrisation is the one notably used in the [Wikipedia article on Rice distribution](https://en.wikipedia.org/wiki/Rice_distribution).

[scipy.stats.rice](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.rice.html) used a different distribution: shape parameter ``b=nu/sigma`` and scale ``sigma``.

The current Rice distribution in pymc inconsistently mixes the two parametrisations: ``nu`` is sometimes the location parameter of the normal distributions, sometimes the shape parameter.

```python
import pymc3 as pm
from scipy import stats
import matplotlib.pyplot as plt
import numpy as np

sigma = 5
nu = 3

u = pm.distributions.Rice.dist(nu, sigma)
v = stats.rice(b=nu/sigma, scale=sigma)

np.random.seed(123)
mean_u = u.mean.eval()

n = 10000
samples_u = u.random(size=n)
samples_v = v.rvs(size=n)

plt.figure()
plt.hist(samples_u, density=True, bins=50, label="pymc3.Rice")
plt.hist(samples_v, density=True, bins=50, label="scipy Rice dist")
plt.axvline(mean_u, label="pymc3.Rice.mean", color="C2")
plt.axvline(np.mean(samples_u), label="empirical mean of pymc3.Rice", color="C3")
plt.legend()

assert np.isclose(mean_u, v.mean())  # pass
```
![figure_1](https://user-images.githubusercontent.com/16596345/49440190-e8696880-f7ba-11e8-88b3-cf6aaf926d4d.png)

I'm writing a PR to fix this.

**Comment by twiecki** [comment]
Good catch!

**Comment by nbud** [comment]
Closed by #3289, see also #3287
